### PR TITLE
:bug: Fix typography menus in left sidebar (assets)

### PR DESCRIPTION
**Pull request by belen-albeza**
- :bug: Fix typography dropdown menus being clipped out in the left sidebar

Fixes https://tree.taiga.io/project/penpot/issue/6355

https://github.com/penpot/penpot/assets/63681/3f579bbd-8412-4f2c-b898-ceeeedb2ebdc

NOTE: this the finished version of https://github.com/penpot/penpot/pull/4013, which I closed.

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/sidebar/assets.scss [hunk 1]
```diff
@@ -7,9 +7,11 @@
 @import "refactor/common-refactor.scss";
 
 .assets-bar {
-  position: relative;
+  display: grid;
   height: 100%;
-  overflow: hidden;
+  grid-template-rows: auto 1fr;
+  // TODO: ñapa :( Fix this! we shouldn't be hardcoding this height
```
Comment (Alotor) [inline frontend/src/app/main/ui/workspace/sidebar/assets.scss]: If you want to keep the comment at least put it in english :smile:

File: frontend/src/app/main/ui/workspace/sidebar/assets.cljs [hunk 2]
```diff
@@ -190,6 +190,7 @@
      [:& (mf/provider cmm/assets-filters) {:value filters}
       [:& (mf/provider cmm/assets-toggle-ordering) {:value toggle-ordering}
        [:& (mf/provider cmm/assets-toggle-list-style) {:value toggle-list-style}
-        [:div {:class (stl/css :libraries-wrapper)}
+        ;; [:div {:class (stl/css :libraries-wrapper)}
```
Comment (Alotor) [inline frontend/src/app/main/ui/workspace/sidebar/assets.cljs]: Remove line

**Review by Alotor — COMMENTED** [review]
Please, review the comments in the scss. Most of the can be deleted.

File: frontend/src/app/main/ui/workspace/sidebar.scss [hunk 3]
```diff
@@ -69,14 +66,25 @@ $width-settings-bar-max: $s-500;
...
-    z-index: $z-index-3;
-    cursor: ew-resize;
-  }
+
   .settings-bar-inside {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 100%;
+
+    height: calc(100vh - $s-52);
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/workspace/sidebar.scss:75]: duplicated line

**Review by EvaMarco — APPROVED** [review]

**Review by Alotor — APPROVED** [review]
Nice work!

**Pull request merged**